### PR TITLE
docs: Add new windows installation tutorials

### PR DESCRIPTION
**Pull request by ahmetb**
Updated Windows installation documentation with newest
screencasts and Chocolatey instructions to install windows
client CLI with these videos:

https://www.youtube.com/watch?v=l026hqjhLRU
https://www.youtube.com/watch?v=TjMU3bDX4vo

Signed-off-by: Ahmet Alp Balkan ahmetalpbalkan@gmail.com

cc: @SvenDowideit @tianon @moxiegirl

:exclamation: @jfrazelle @crosbymichael we should pick this into 1.6 release branch. Thanks! :exclamation:

### CODE REVIEW

File: docs/sources/installation/windows.md [hunk 1]
```diff
@@ -43,6 +43,26 @@ is developed, you can launch only Linux containers from your Windows machine.
 
 ![](/installation/images/windows-boot2docker-start.png)
 
+##  Installing Docker client (without Boot2Docker)
```
Comment (moxiegirl) [inline docs/sources/installation/windows.md]: @ahmetalpbalkan We are moving to a policy where we don't document other project's installers or package manager installs.  There are several reasons for this:

* It puts a burden on us to test 
* It puts a burden on us to update

Happy to put a mention in a references section.

## References

If you have Docker hosts running and if you don't wish to do a Boot2Docker installation, you can install the `docker.exe` using unofficial Windows package manager Chocolately. For information on how to do this, see (instructions at [chocolatey.org](http://chocolatey.org)).

Comment (ahmetalpbalkan) [inline docs/sources/installation/windows.md]: @moxiegirl I believe I can do that.

Comment (ahmetalpbalkan) [inline docs/sources/installation/windows.md]: @moxiegirl fixed, please review.

**Comment by moxiegirl** [comment]
Thank you @ahmetalpbalkan! LGTM

**Comment by jfrazelle** [comment]
LGTM

**Comment by ahmetalpbalkan** [comment]
@jfrazelle assuming will be cherrypicked. thanks a lot for reviewing! Looks good here http://docs.master.dockerproject.com/installation/windows/

**Comment by jfrazelle** [comment]
cherrypicked

**Pull request merged**